ArticleTemplates/assets/js/bootstraps/common.js

```diff
@@ -187,45 +187,45 @@ define([
             };
         },
 
-        showTabs: function () {
+        showTabs: function (root) {
             // Set up tab events, show only first child
-            $('.tabs a').each(function (el, i) {
-                var tabGroup = el.getAttribute('href');
-                var obj;
-                var AdStatus;
+            var tabContainer = $('.tabs');
+            var tabs = $('a',tabContainer);
+
+            tabs.each(function(tab,i) {
                 if (i > 0) {
-                    $(tabGroup).hide();
+                    $(tab.getAttribute('href')).hide();    
                 }
-                bean.on(el, 'click', function (e) {
-                    e.preventDefault();
+            });
+
+            bean.on(tabContainer[0], 'click', 'a', function (e) {
+
+                e.preventDefault();                
+                var tab = $(this);
 
+                if( tab.attr("aria-selected") !== 'true' ) {
+                 
                     $($('[aria-selected="true"]').attr('href')).hide();
```

Comment:
You haven't touched this part, but maybe this selector can be cached, `$('[aria-selected="true"]')`